Firmware/Marlin_main.cpp

```diff
@@ -321,6 +321,7 @@ uint8_t saved_bed_temperature; //!< Bed temperature
 bool saved_extruder_relative_mode;
 uint8_t saved_fan_speed = 0; //!< Print fan speed
 //! @}
+bool uvlo_auto_recovery_ready = false;
```

Comment:
Nitpick: Please make this `static bool` :)

Reply:
Done